app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,10 @@
+class PostsController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    @posts = Post.where(author: @user)
+  end
+
+  def show
+    @post = Post.find(params[:id])
+  end
+end
```

Comment:
**OPTIONAL** - Please note that according to the project requirements, you are not required to have any logic in your controller actions, this would be done in a later milestone. I suggest leaving your actions empty 👍 👍